README.md

```diff
@@ -435,6 +355,347 @@ One App can be started via docker or if built from source by running `node lib/s
 
 <!-- TODO talk about prometheus and logging schema here -->
 
+## 🎛️ API
+
+### Overview
+
+#### [Modules](#modules)
+
+**Holoron Modules** or "Modules" are self contained Web Experiences that consist of React Components with Redux-compatible Reducers and Actions. In practice, Modules are developed and bundled in isolation to one another. The One App Server uses a Module Map containing URLs to Module bundles (e.g. `my-module.browser.js`) to load and serve bundles upon request. When the Server receives an HTTP request, it renders one or more Modules on the Server. Similar to React Components, Modules are composable (e.g. Modules may load other Modules). The first or entrypoint Module is called the "Root Module". The Root Module loads other "Child Modules". Overall, this development pattern in One App may be characterized as the [Micro Front End](https://martinfowler.com/articles/micro-frontends.html) pattern.
+
+**API**
+* [loadModuleData](#loadmoduledata)
```

Comment:
some of the anchors contain a hyphen and others don't? 

`* [loadModuleData](#loadmoduledata)`

`#### [Server](#-server)`

Reply:
If there's an emoji, we need to use a dash to prefix the anchor, I've observed.